lib/presentation/view/common/common_view.dart

```diff
@@ -44,6 +44,10 @@ class CommonView {
   }
 
   Widget buildResultCountRow(BuildContext context, int count) {
+    if (count == 0) {
```

Comment:
You updated here but haven't used this function in the screens (my space, received share, shared space). 
If you use this, you don't need to check the length of state's list more :)

Reply:
yh u are right i'll update other too

Reply:
> How about the UI whenever no result matched? @imGok

![image](https://user-images.githubusercontent.com/48354990/133292169-bead09af-59f1-41a8-9b70-e6c6bc745f34.png)